docroot/modules/custom/mass_media/tests/src/ExistingSite/MediaBulkActionTest.php

```diff
@@ -100,11 +99,16 @@ private function createMediaFile() {
   private function runTestSteps($action) {
     $this->createMediaFile();
     $this->drupalGet('admin/ma-dash/documents');
-    $this->getCurrentPage()->find('css', 'select[name="action"]')->selectOption($action);
-    $this->getCurrentPage()->find('css', 'input[name="views_bulk_operations_bulk_form[0]"]')->check();
-    $this->getCurrentPage()->pressButton('Apply to selected items');
-    $this->getCurrentPage()->pressButton('Execute action');
-    $this->assertSession()->waitForText('Your changes have been successfully made.');
+    $edit = [
+      'action' => $action,
+      'views_bulk_operations_bulk_form[0]' => TRUE,
+    ];
+    $this->submitForm($edit, $this->t('Apply to selected items'), 'views-form-all-documents-page-1');
+    $this->submitForm([], $this->t('Execute action'), 'views-bulk-operations-confirm-action');
+    $page = $this->getSession()->getPage();
+    $page->waitFor(3, function () use ($page) {
```

Comment:
I could not quite avoid adding a wait somewhere in these tests due to the batch page. If the tests prove to be flaky again, we can increase this number.